clients/java/src/main/java/io/camunda/client/impl/CamundaClientBuilderImpl.java

```diff
@@ -87,7 +99,7 @@ public final class CamundaClientBuilderImpl
       Collections.singletonList(CommandWithTenantStep.DEFAULT_TENANT_IDENTIFIER);
   private int jobWorkerMaxJobsActive = 32;
   private int numJobWorkerExecutionThreads = 1;
-  private String defaultJobWorkerName = DEFAULT_JOB_WORKER_NAME;
+  private String defaultJobWorkerName = DEFAULT_JOB_WORKER_NAME_VAR;
```

Comment:
💭 Just curious why we decided to rename this variable?

Reply:
Because otherwise it will conflict with this import `import static io.camunda.client.ClientProperties.DEFAULT_JOB_WORKER_NAME;`